src/libraries/System.Private.CoreLib/src/System/IO/File.cs

```diff
@@ -159,6 +209,13 @@ public static FileStream Open(string path, FileMode mode, FileAccess access, Fil
             return new FileStream(path, mode, access, share);
         }
 
+#if NET6_0_OR_GREATER
+        public static FileStream Open(string path, FileMode mode, FileStreamOptions options)
+        {
+            return new FileStream(path, Override(options, mode));
```

Comment:
We've made a mistake in the API review meeting. This method should not be accepting `mode` as `FileStreamOptions` already allows for specifying that. https://github.com/dotnet/runtime/issues/24698#issuecomment-851386325

```suggestion
        public static FileStream Open(string path, FileStreamOptions options)
        {
            return new FileStream(path, options);
```

Reply:
Fixed.